plugins/BEdita/Core/tests/Fixture/ObjectTagsFixture.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2019 ChannelWeb Srl, Chialab Srl
```

Comment:
```suggestion
 * Copyright 2022 ChannelWeb Srl, Chialab Srl
```
minor, current year for copyright